.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test DBOS eslint plugin
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
We should use node 20: the SDK already declares 20 in its package.json and we have been steadily moving toward 20 in the cloud

Reply:
We should certainly support and encourage 20.  The question I suppose is whether we allow 18 also for CY 24 (certainly not for CY 25).  Lets discuss at the meeting, if there is no interest in 18 I will bump it to 20.

(I chose 18 because the SDK itself chose 18)